apps/condo/domains/ticket/components/BaseTicketForm/index.tsx

```diff
@@ -254,7 +246,9 @@ const TicketAssignments = ({ validations, organizationId, propertyId, disableUse
                     >
                         <GraphQlSearchInput
                             formatLabel={formatUserFieldLabel}
-                            search={searchEmployeeUser(organizationId, whoCanBeAssignedAsExecutor)}
+                            search={searchEmployeeUser(organizationId, ({ role }) => (
+                                get(role, 'canBeAssignedAsExecutor', false)
```

Comment:
(not related to this task) could you please add the doc for `canBeAssignedAsExecutor` role field? It's not so clear to understand what's going on.

Reply:
Added in `790eceb6`